src/electionguard/ballot.py

```diff
@@ -877,9 +877,7 @@ def make_ciphertext_ballot(
     if previous_ballot_code is None:
```

Comment:
This linen should also be `code_seed`.

 I would do a search for `previous_ballot_code`

Reply:
sorry overlooked those